src/UI/templates/default/Dropdown/tpl.standard.html

```diff
@@ -1,4 +1,4 @@
-<div class="dropdown"><button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown" <!-- BEGIN with_id --> id="{ID}"<!-- END with_id --> aria-haspopup="true" aria-expanded="false">{LABEL} <span class="caret"></span></button>
+<div class="{CLASSNAME}"><button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown" <!-- BEGIN with_id --> id="{ID}"<!-- END with_id --> aria-haspopup="true" aria-expanded="false">{LABEL} <span class="caret"></span></button>
```

Comment:
Bad. Class names should appear explicit in templates. You may use something like 
`<!-- BEGIN with_dropdown_class -->class="dropdown"<!-- END with_dropdown_class -->`